File: abecedario.py
xaxis = range(int(input('ingrese el numero de caracteres que desea ver horizontalmente:\n')))  # numero de caracteres que se mostraran horizontalmente
yaxis = range(int(input('ingrese el numero de caracteres que desea ver verticalmente:\n')))  # numero de caracteres que se mostrara verticalmente
letters = 'abcdefghijklmnopqrstuvwxyz'
letters = letters[:letters.find(input('ingresa la letra limite:\n').lower()) + 1]  # String que contiene desde la a la letra ingresada
changin = letters  # variable que guarda copia a letters para luego ser modificado
lon = len(letters) - 1  # variable que guarda el numero de subindices de letters
white, rev = 0, False  # white determina el numero de espacios condicionada por rev
for y in yaxis:
	limit = 0
	changin = letters[:white + 1] + (lon - white) * ' '
	for x in xaxis:
		if limit == (lon + 1):
			changin = changin[-1::-1]
			limit = 1
		print(changin[limit], end='')
		limit += 1
	print('')
	if rev:
		white -= 1
	else:
		white += 1
	if white == lon:
		rev = True
	if white == 0:
		rev = False
	changin = letters

File: abecedario.cpp
//Librerias
#include <iostream>
#include <string>

//Namespace
using namespace std;


int main (){
    // variables
        string letters = "abcdefghijklmnopqrstuvwxyz";
        int xaxis;
        int yaxis;
        string letter;
    cout << "Ingresa el numero de caracteres que deseas ver horizontalmente:\n";
    cin >> xaxis;
    cout << "Ingresa el numero de caracteres que deseas ver verticalmente:\n";
    cin >> yaxis;
    cout << "Ingresa La letra limite:" << endl;
    cin >> letter;
    size_t pos = letters.find(letter) + 1;
    letters = letters.substr(0, pos);
    string changin = letters;
    int white = 0;
    bool rev = false;

    cout << changin;
    for (int i = 0; i < yaxis; i++){
        int limit = 0;
        
    }

    return 0;
}

